website/content/docs/enterprise/license/faq.mdx

```diff
@@ -61,13 +68,15 @@ As Consul Enterprise approaches the expiration date, warnings will be issued in
 
 ## Q: Does this affect client agents?
 
-For existing clusters, if the clients agents are using ACLs and have a valid token, then they will be able to retrieve the license from the server.
+There are upgrade requirements that affect Consul Enterprise clients.
+Please review the [upgrade requirements](faq#q-what-are-the-upgrade-requirements) documentation.
+
 If the client agents are not using ACLs, then the client agents will be need to have the license on-disk (auto-loading) or as an environment variable.
 For new Consul clusters using Consul 1.10.0+ent, customers must ensure that there is a valid license on-disk (auto-loaded) or as an environment variable.
```

Comment:
I think this second sentence "For new clusters using .." can be removed, since it doesn't apply to *new* 1.10.0 clients and this sentence is in the client agent section, also the latest commit added the upgrade requirements above and linked to that already..